### ISSUE TITLE
Got error when trying to install @webex/widgets

### ISSUE THREAD

**Issue by Bhavik1363** [webex/widgets#214]
I am trying to install the package from the following command,
npm i @webex/widgets

It shows the package installed successfully.

But when importing the WebexMeetingsWidget component into the app.js it throws almost 82 errors.

![Screenshot from 2023-09-13 16-45-19](https://github.com/webex/widgets/assets/47604057/8790d13c-82b3-465b-8bc4-0e3aee4701aa)

I am using the React application with react^18.2.0

**Comment by Junior743** [comment]
I have a similar problem, when a try to run: npx install-peerdeps @webex/widgets

I received this result:
![image](https://github.com/webex/widgets/assets/19689949/6ce9f2ab-a896-4986-a63c-c14a3ef4cb04)

I think the problem is the react version. @webex|widgets are trying to install react 17, but I already using react 18.

**Comment by carrubbasamuel** [comment]
I have the same problem 
- Next version: 14.x.x
- react :18.2.0

**Comment by sreenara** [comment]
Hey folks. Thanks for reporting these issues and providing feedback. We'll get someone to look into this soon.
@Kesari3008 do any of these errors seem familiar and similar to what we observed as we tried to upgrade our tooling?

**Comment by Kesari3008** [comment]
> Hey folks. Thanks for reporting these issues and providing feedback. We'll get someone to look into this soon. @Kesari3008 do any of these errors seem familiar and similar to what we observed as we tried to upgrade our tooling?

@sreenara Yes we have seen a similar error while upgrading our tooling but only once. Hence I cannot say for sure why it happens. One possible reason can be mismatch of React version as Widgets currently uses React 17. One way to resolve this is I remember that I installed querystring, crypto, os-browserify modules to resolve these polyfill errors. As a workaround, that's something which can be tried. In the meantime, I will check try this out to on my end again and find it's proper solution.

**Comment by sreenara** [comment]
@Bhavik1363 @Junior743 @carrubbasamuel are you'll able to use React 17.x as a workaround for now? We will be working towards upgrading widgets to React 18 in the future.
